sandy/core/endf6.py

```diff
@@ -56,6 +65,86 @@
 pd.options.display.float_format = '{:.5e}'.format
 
 
+def get_tsl_index(library):
+    """
+    Obtain the index information available in the library web page.
+
+    Parameters
+    ----------
+    library : `str`
+        nuclear data library. Available libraries are:
+        for 'tsl'
+            * `'endfb_71'`
+            * `'jeff_33'`
+            * `'endfb_80'`
+            * `'jendl_40u`
+
+    Raises
+    ------
+    ValueError
+        if library is not among available selection.
+
```

Comment:
You forgot the `Return`

Reply:
There is no return, only print data on the screen